.cloudbuild/ci/integration-tests.yaml

```diff
@@ -11,7 +11,7 @@ steps:
 
   # Set up BCC headers in a volume to replace the incompatible system-level 
   # headers
-  - name: us-docker.pkg.dev/ci-account/teleport/buildbox-root:teleport6.2-v0.1.0
+  - name: quay.io/gravitational/teleport-buildbox:go1.15.5
```

Comment:
Go 1.15 latest patch is [1.15.15](https://go.dev/dl/#go1.15.15), should we upgrade the image? Maybe in a separate PR?

(Btw, I really appreciate the clean buildbox labels, makes spotting stuff like that trivial.)